Client/Frontend/Toolbar+URLBar/TabLocationView.swift

```diff
@@ -110,15 +114,23 @@ class TabLocationView: UIView, FeatureFlaggable {
         shareButton.contentHorizontalAlignment = .center
         shareButton.accessibilityIdentifier = AccessibilityIdentifiers.Toolbar.shareButton
         shareButton.accessibilityLabel = .TabLocationShareAccessibilityLabel
+        shareButton.showsLargeContentViewer = true
+        shareButton.largeContentImage = .templateImageNamed(ImageIdentifiers.share)
+        shareButton.largeContentTitle = .TabLocationShareButtonAccessibilityLabel
     }
 
     private lazy var shoppingButton: UIButton = .build { button in
+        let image = UIImage(named: StandardImageIdentifiers.Large.shopping)
+
         button.addTarget(self, action: #selector(self.didPressShoppingButton(_:)), for: .touchUpInside)
         button.isHidden = true
-        button.setImage(UIImage(named: StandardImageIdentifiers.Large.shopping)?.withRenderingMode(.alwaysTemplate), for: .normal)
+        button.setImage(image?.withRenderingMode(.alwaysTemplate), for: .normal)
         button.imageView?.contentMode = .scaleAspectFit
         button.accessibilityLabel = .TabLocationShoppingAccessibilityLabel
         button.accessibilityIdentifier = AccessibilityIdentifiers.Toolbar.shoppingButton
+        button.showsLargeContentViewer = true
+        button.largeContentTitle = .TabLocationShoppingAccessibilityLabel
+        button.largeContentImage = image
```

Comment:
I reused an existing accessibility label. Should I create a separate localized string for the `largeContentTitle`?